deploy/minikube/release_sanity_test.go

```diff
@@ -70,10 +72,43 @@ func TestReleasesJson(t *testing.T) {
 			actualSha, err := getShaFromURL(getDownloadURL(r.Name, platform))
 			if err != nil {
 				t.Errorf("Error calcuating SHA for %s-%s. Error: %s", r.Name, platform, err)
+				continue
 			}
 			if actualSha != sha {
 				t.Errorf("ERROR: SHA does not match for version %s, platform %s. Expected %s, got %s.", r.Name, platform, sha, actualSha)
+				continue
```

Comment:
Yeah, t.Errorf will just continue the loop so we'd see a bunch of errors here instead of just one.